xbmc/LangInfo.cpp

```diff
@@ -661,17 +661,26 @@ bool CLangInfo::SetLanguage(std::string language /* = "" */, bool reloadServices
   if (language.empty())
     language = CServiceBroker::GetSettingsComponent()->GetSettings()->GetString(CSettings::SETTING_LOCALE_LANGUAGE);
 
+  auto& addonMgr = CServiceBroker::GetAddonMgr();
   ADDON::AddonPtr addon;
-  if (!CServiceBroker::GetAddonMgr().GetAddon(language, addon, ADDON::ADDON_RESOURCE_LANGUAGE, false))
+  if (!addonMgr.GetAddon(language, addon, ADDON::ADDON_RESOURCE_LANGUAGE, true))
```

Comment:
I would add a comment here. `// Find the chosen language add-on if it's enabled`. Cause no one will know what the true refers to.